sw/device/silicon_owner/bare_metal/bare_metal_start.S

```diff
@@ -151,9 +151,7 @@ _start_boot:
    *
    * The lowest two bits should be `0b01` to ensure we use vectored interrupts.
    */
-  la   t0, _interrupt_vector
-  andi t0, t0, -4
-  ori  t0, t0, 0b01
```

Comment:
I think you need ori instruction , because of that:
`The lowest two bits should be 0b01 to ensure we use vectored interrupts.`

Reply:
Please scratch that, I missed that there is +1.